portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,22 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+//Causes the bubble to slide down the page.
+function bubbleFall() {
+    //Find the bubble.
+    var elem = document.getElementById('bubble');
```

Comment:
Can we change var to const or let? https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
Yes! My thanks again. I will wait until I have checked style more thoroughly before future commits.